Gemfile.lock

```diff
@@ -334,45 +347,46 @@ GEM
       ttfunk
     php-serialization (0.5.3)
       racc
-    pkg-config (1.1.7)
-    polyamorous (1.3.0)
+    polyamorous (1.3.1)
       activerecord (>= 3.0)
     powerpack (0.1.1)
-    pry (0.10.3)
+    pry (0.10.4)
       coderay (~> 1.1.0)
       method_source (~> 0.8.1)
       slop (~> 3.4)
     racc (1.4.14)
-    rack (1.6.4)
-    rack-dev-mark (0.7.4)
+    rack (2.0.1)
+    rack-dev-mark (0.7.5)
       rack (>= 1.1)
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.2.6)
-      actionmailer (= 4.2.6)
-      actionpack (= 4.2.6)
-      actionview (= 4.2.6)
-      activejob (= 4.2.6)
-      activemodel (= 4.2.6)
-      activerecord (= 4.2.6)
-      activesupport (= 4.2.6)
+    rails (5.0.0)
```

Comment:
__[brakeman]__
* Rails 5.0.0 content_tag does not escape double quotes in attribute values (CVE-2016-6316). Upgrade to 5.0.0 [:ref](https://groups.google.com/d/msg/ruby-security-ann/8B2iV2tPRSE/JkjCJkSoCgAJ)

<!-- Comment by [SideCI](https://sideci.com) -->